content/02.main-text.md

```diff
@@ -317,6 +321,34 @@ While Manubot does require technical expertise, the benefits are manyfold.
 Specifically, Manubot demonstrates a system for publishing that is transparent, reproducible, immediate, permissionless, versioned, automated, collaborative, open, linked, provenanced, decentralized, hackable, interactive, annotated, and free of charge.
 These attributes empower integrating Manubot with an ecosystem of other community-driven tools to make science as open and collaborative as possible.
 
+## Future Manubot Enhancements
+
+Manubot is still under active development, and we envision major changes in its design and dependencies going forward.
```

Comment:
Maybe `major changes` -> `major improvements`?

Reply:
"Major improvements in dependencies" doesn't make sense to me.

Reply:
You're right, leave this as is.